### PR TITLE
Clarify when randomization happens for researchers: Fixes DOC-1182

### PR DESCRIPTION
**Pull request by lamagnifica**
@shnayder, see tracking_logs.rst: you mentioned this as a pain point for researchers.
@catong, your change (and another tweak) to create_video.rst
@mhoeber , fixed a typo in the glossary for content experiments
@WatsonEmily you asked to be tagged!

### CODE REVIEW

File: en_us/data/source/internal_data_formats/tracking_logs.rst [hunk 1]
```diff
@@ -2620,9 +2620,11 @@ A/B Testing Events
 
 Course authors can configure course content to present modules that contain
 other modules. For example, a parent module can include two child modules with
-content that differs in some way for comparison testing. When a student
-navigates to a module that is set up for A/B testing in this way, the student
-is randomly assigned to a group and shown only one of the child modules.
+content that differs in some way for comparison testing. When a student first
+visits a course component that is set up to be the parent module of an A/B
```
Comment (mhoeber) [inline en_us/data/source/internal_data_formats/tracking_logs.rst]: can we also use "content experiments" as main term everywhere?

**Comment by mhoeber** [comment]
thx.  one question, otherwise :+1:

**Comment by lamagnifica** [comment]
@mhoeber please see effects on sql_schema.rst in addition to tracking_logs.rst

**Comment by mhoeber** [comment]
:+1:

File: en_us/data/source/internal_data_formats/tracking_logs.rst [hunk 2]
```diff
@@ -2614,19 +2614,22 @@ the response.
...
+========================================
+Testing Events for Content Experiments
+========================================
 
 Course authors can configure course content to present modules that contain
-other modules. For example, a parent module can include two child modules with
-content that differs in some way for comparison testing. When a student
-navigates to a module that is set up for A/B testing in this way, the student
-is randomly assigned to a group and shown only one of the child modules.
+other modules. Content experiments, also known as A/B or split test, use this
```
Comment (catong) [inline en_us/data/source/internal_data_formats/tracking_logs.rst]: "known as A/B or split test_s_"? (missing s)

File: en_us/course_authors/source/creating_content/create_video.rst [hunk 3]
```diff
@@ -86,7 +86,7 @@ When you create your video, edX recommends the following compression specs.
      - AAC 44.1 / 192 kbps
      - AAC 44.1 / 192 kbps
 
-.. note:: In general, export at the same frame rate used to shoot the media file. For example, in PAL-based countries, export at 25 fps.
+.. note:: In general, export at the same frame rate used to create the media file. For example, in countries that use the PAL system, export at 25 fps.
```
Comment (catong) [inline en_us/course_authors/source/creating_content/create_video.rst]: Still nitpicky, sorry -- does the PAL system use 25 fps?  We're not actually spelling out that it does. What if it changes in future? Hard to catch.

Comment (catong) [inline en_us/course_authors/source/creating_content/create_video.rst]: Probably too wordy, but I was looking for something like "For example, because the PAL system uses 25 fps, in countries that use PAL, export at 25 fps."

**Comment by lamagnifica** [comment]
changes to create_video.rst are ok per Rachel.

**Comment by WatsonEmily** [comment]
:+1:

**Comment by lamagnifica** [comment]
@shnayder please review, this is waiting only on your input.

File: en_us/data/source/internal_data_formats/tracking_logs.rst [hunk 4]
```diff
@@ -2614,19 +2614,22 @@ the response.
...
 
 Course authors can configure course content to present modules that contain
-other modules. For example, a parent module can include two child modules with
-content that differs in some way for comparison testing. When a student
-navigates to a module that is set up for A/B testing in this way, the student
-is randomly assigned to a group and shown only one of the child modules.
+other modules. Content experiments, also known as A/B or split tests, use this
+structure. For example, a parent module can include two child modules that
+contain content that differs in some way for comparison testing. When a student
+first visits a course component that is set up to be the parent module of a
```
Comment (shnayder) [inline en_us/data/source/internal_data_formats/tracking_logs.rst]: It's not necessarily when they first visit the component. It's the first time that data about any component connected to that experiment group is needed--this could e.g. happen when the student loads their progress page. The only guarantee is that it will happen before the student sees any of the tested content. 

To get precise data on who in each group actually saw the tested components, researchers should use the behavior events--problem checks, video views, etc.

**Comment by lamagnifica** [comment]
@shnayder, I appreciate the course correction! Reorganized somewhat to add your info. lemme know

**Comment by shnayder** [comment]
Looks good. Note for future: it may be interesting to work with one of the research teams that's actually doing this to write a case study of how to really analyze A/B test results in detail. It gets complex....

**Comment by lamagnifica** [comment]
Making a story! that would be a great project.
DOC-1321
